Patch origin: in-house
Patch status: Solaris-specific; not suitable for upstream

diff --git a/config.layout b/config.layout
index a29351e..aca0c93 100644
--- a/config.layout
+++ b/config.layout
@@ -226,26 +226,26 @@
     proxycachedir: ${localstatedir}/proxy
 </Layout>
 
-#   Solaris 8 Layout
+#   Solaris Layout
 <Layout Solaris>
-    prefix:        /usr/apache
+    prefix:        /usr/apache2/2.4
     exec_prefix:   ${prefix}
     bindir:        ${exec_prefix}/bin
     sbindir:       ${exec_prefix}/bin
     libdir:        ${exec_prefix}/lib
     libexecdir:    ${exec_prefix}/libexec
     mandir:        ${exec_prefix}/man
-    sysconfdir:    /etc/apache
-    datadir:       /var/apache
-    installbuilddir: ${datadir}/build
+    sysconfdir:    /etc/apache2/2.4
+    datadir:       /var/apache2/2.4
+    installbuilddir: ${prefix}/build
     errordir:      ${datadir}/error
     iconsdir:      ${datadir}/icons
     htdocsdir:     ${datadir}/htdocs
-    manualdir:     ${datadir}/manual
+    manualdir:     ${prefix}/manual
     cgidir:        ${datadir}/cgi-bin
     includedir:    ${exec_prefix}/include
     localstatedir: ${prefix}
-    runtimedir:    /var/run
+    runtimedir:    /var/run/apache2/2.4
     logfiledir:    ${datadir}/logs
     proxycachedir: ${datadir}/proxy
 </Layout>
